### PR TITLE
[Docs]: Updating Docs for Contributing to Model Relationships

### PR DESCRIPTION
**Pull request by pandeyshubham03**
**Notes for Reviewers**

This PR fixes #
1. Removes Links from the documentation
2. Adds a description for creating a policy for evaluation

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c381e5510276d3feb3ca6e1f7b6a27e557eace50 at: https://660426d1320eb9109f46056d--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-model-relationships.md [hunk 1]
```diff
@@ -222,70 +223,4 @@ Scope is defined and controlled via the `selectors` [Selectors](#structure-of-se
    - No relationship type (Kind) is inherently more important than the next one, so will not be any case
      of conflict
 
-## Relationship Types
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-model-relationships.md:225]: This is some of the most explanatory content in the page. Why is this being removed, @pandeyshubham03? @ritiksaxena124 @nwanduka?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/project/contributing/contributing-model-relationships.md (hunk 1 above)
Comment (pandeyshubham03) [reply docs/pages/project/contributing/contributing-model-relationships.md:225]: We already have the details for the Relationships kinds and subtypes in line 30. This part is a repetition.

File: docs/pages/project/contributing/contributing-model-relationships.md [hunk 2]
```diff
@@ -21,33 +21,34 @@ Relationships within Model play a crucial role in establishing concrete visualis
    - [Binding]({{ site.baseurl }}/assets/img/meshmodel/relationships/binding_relationship.png)
    - [Edge]({{ site.baseurl }}/assets/img/meshmodel/relationships/mount_edge_relationship.png)
 
-4. Prospose the appropriate relationship type, using one of the predefined set of relationship types or suggest a new relationship where an existing type does not fit.
+4. Propose the appropriate relationship type, using one of the predefined set of relationship types, or suggest a new relationship where an existing type does not fit.
 5. Create a Relationship Definition (yaml).
-6. Create a policy for evaluation of the relationship (rego). See [examples](https://github.com/meshery/meshery/tree/master/server/meshmodel/policies/).
+6. Create a policy for evaluation of the relationship (rego).
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-model-relationships.md:26]: We shouldn't be asking for creating, @MUzairS15

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/project/contributing/contributing-model-relationships.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-model-relationships.md:225]: The repetition up top is the worse of the two.

**Comment by l5io** [comment]
🚀 Preview for commit 73b2fe4ccdac19325a41f978f4e55b7815d57afb at: https://660582fdb881001bae09e620--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 87d6692b6cda7910c442d71623c92c443d55c044 at: https://66058d06dc0d7824f55f740c--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@RipulHandoo make sure that the inventory - wallet (e.g. configmap) and the inventory - badge (e.g. Wasm filter) is captured in the examples.

**Pull request merged**